### PR TITLE
Feature/account activation

### PR DESCRIPTION
**Pull request by virgotopia**

### CODE REVIEW

File: src/sharepoint/activate-account/sharepoint-activate-account.controller.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            this.alerter = Alerter;
+            this.sharepointService = MicrosoftSharepointLicenseService;
+            this.$stateParams = $stateParams;
+            this.$scope = $scope;
+        }
+
+        $onInit () {
+            this.account = this.$scope.currentActionData;
+
+            this.$scope.submit = () => {
```
Comment (FredericEspiau) [inline src/sharepoint/activate-account/sharepoint-activate-account.controller.js]: Should be in a named function

File: src/sharepoint/sharepoint.service.js [hunk 2]
```diff
@@ -283,6 +283,21 @@
...
+            /**
+             * Activate a sharepoint on an account
+             * @param {string} serviceName
+             * @param {string} userPrincipalName
+             */
+            activateSharepointOnAccount (serviceName, account) {
+                return this.OvhHttp
+                    .post(`/msServices/${serviceName}/account/${account}/sharepoint/configure`, {
+                        rootPath: "apiv6",
+                        data: { },
```
Comment (FredericEspiau) [inline src/sharepoint/sharepoint.service.js]: is that useful ?

**Comment by antleblanc** [comment]
@virgotopia You should target `develop` instead of `master` branch. @jleveugle Am I right?

**Comment by jleveugle** [comment]
@FredericEspiau 

I close it due to inactivity, please reopen it if needed

**Pull request closed without merging**